helper/src/main/java/me/lucko/helper/utils/CommandMapUtil.java

```diff
@@ -26,24 +26,17 @@
 package me.lucko.helper.utils;
 
 import com.google.common.base.Preconditions;
-
 import org.bukkit.Bukkit;
-import org.bukkit.command.Command;
-import org.bukkit.command.CommandExecutor;
-import org.bukkit.command.CommandMap;
-import org.bukkit.command.PluginCommand;
-import org.bukkit.command.SimpleCommandMap;
-import org.bukkit.command.TabCompleter;
+import org.bukkit.command.*;
```

Comment:
This is the only instance of a wildcard import that I've seen, but you should ensure you're not accidentally putting them anywhere else, as well.

Wildcard imports are a cancer to open-source, multi-contributor projects.